### PR TITLE
Liveness probe added in ContainerCard component

### PR DESCRIPTION
**Pull request by deepan10**
## Hey, I just made a Pull Request!

Only the `Readiness` probe is showing in the ContainerCard component in Kubernetes PodDrawer, this PR will enable the `Liveness Probe` too in PodDrawer.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by deepan10** [comment]
Hello @mclarke47 
Continuing the discussion from https://github.com/backstage/backstage/pull/24395

Here we are just displaying what the POD has, it's up to the developer to decide which probe to use. 

Also, I could see the upcoming release supports both the `Readiness` & `Liveness` health endpoint on Backstage.
https://backstage.io/docs/releases/v1.30.0-next.0-changelog/#patch-changes-6
https://backstage.io/docs/reference/backend-plugin-api.roothealthservice

Would you still like this to be customizable?

**Comment by mclarke47** [comment]
Can we add a condition so this only gets added if the probe is specified?

**Comment by deepan10** [comment]
Sure, will do that 👍

File: plugins/kubernetes-react/src/components/Pods/PodDrawer/ContainerCard.tsx [hunk 1]
```diff
@@ -41,6 +41,16 @@ const getContainerHealthChecks = (
       'no restarts': containerStatus.restartCount === 0,
     };
   }
+  if (containerSpec && containerSpec?.livenessProbe !== undefined) {
```
Comment (mclarke47) [inline plugins/kubernetes-react/src/components/Pods/PodDrawer/ContainerCard.tsx:44]: Can we avoid the duplication of the conditions? for example, define the checks in the return object, if the liveness probe is defined add the new check to the object, return the object

**Review by mclarke47 — APPROVED** [review]

**Pull request merged**